pom.xml

```diff
@@ -117,7 +110,6 @@
       <plugins>
         <plugin>
           <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
It seems the new [Parent POM](https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L481) forces to use Maven 3 and here, we have to use Maven 2., right?